internal/uvm/network.go

```diff
@@ -592,22 +592,30 @@ func (uvm *UtilityVM) addNIC(ctx context.Context, id string, endpoint *hns.HNSEn
 		}
 	} else {
 		// Verify this version of LCOW supports Network HotAdd
+		s := &guestresource.LCOWNetworkAdapter{
+			NamespaceID:     endpoint.Namespace.ID,
+			ID:              id,
+			MacAddress:      endpoint.MacAddress,
+			IPAddress:       endpoint.IPAddress.String(),
+			PrefixLength:    endpoint.PrefixLength,
+			GatewayAddress:  endpoint.GatewayAddress,
+			DNSSuffix:       endpoint.DNSSuffix,
+			DNSServerList:   endpoint.DNSServerList,
+			EnableLowMetric: endpoint.EnableLowMetric,
+			EncapOverhead:   endpoint.EncapOverhead,
+		}
+		if v6 := endpoint.IPv6Address.To16(); v6 != nil && !v6.IsUnspecified() {
+			// if the address is empty or invalid, endpoint.IPv6Address.String will be "<nil>"
+			// since we should be guranteed an IPv4, but not a v6, skip invalid v6 address
```

Comment:
should we add a log statement if we skip a v6 address?

Reply:
I added a statement for when we do add v6, since its probably very rare that it would be added, we would be logging every time its skipped, which feels excessive